src/trivia/libraries/problems/problems.go

```diff
@@ -3,6 +3,7 @@ package problems
 import (
 	"encoding/csv"
 	"flag"
+	"fmt"
```

Comment:
something I missed in this package last time is that the CSV filename is parsed right here

that's confusing for someone working on the program - it would be better to parse the filename in the Trivia() function so that external inputs are obvious. Then the filename can be passed down into  GetProblems.

Reply:
a2a5ff1